File: program/convert-weeks-to-days/convert_weeks_to_days.pl
print "Enter the no of weeks : ";
my $weeks = <STDIN> ;

my $res = week2Days($weeks);
print "Days conversion for the given weeks : $res days";

sub week2Days
{
    my ($x) = @_;
    my $res = $x * 7;
    return $res ;   
}

File: program/convert-weeks-to-days/convertWeeksToDays.scala
import scala.io.StdIn._;

object weeksToDays {
  def main(args:Array[String]):Unit={
    println("Enter the number of weeks : ");
    var weeks = readLine();
    var days = weeks.toInt * 7;
    println(weeks+" weeks is "+days+" days");
  }
}
